brouter-core/src/main/java/btools/router/RoutingEngine.java

```diff
@@ -73,12 +81,19 @@ public class RoutingEngine extends Thread {
 
   public RoutingEngine(String outfileBase, String logfileBase, File segmentDir,
                        List<OsmNodeNamed> waypoints, RoutingContext rc) {
+    this(0, outfileBase, logfileBase, segmentDir,
+      waypoints, rc);
+  }
+
+  public RoutingEngine(int engineMode, String outfileBase, String logfileBase, File segmentDir,
+                       List<OsmNodeNamed> waypoints, RoutingContext rc) {
```

Comment:
I think the first parameters of the constructors should be the same and engineMode should be appended.